packages/nodes-base/nodes/Google/BigQuery/v2/actions/database/executeQuery.operation.ts

```diff
@@ -3,7 +3,7 @@ import type { IExecuteFunctions } from 'n8n-core';
 import type { IDataObject, INodeExecutionData, INodeProperties } from 'n8n-workflow';
 
 import { NodeOperationError, sleep } from 'n8n-workflow';
-import { updateDisplayOptions } from '../../../../../../utils/utilities';
+import { getResolvables, updateDisplayOptions } from '../../../../../../utils/utilities';
```

Comment:
We can define a path alias for `nodes-base` to avoid having an excessively long import path whenever a node needs a util.